stable/elasticsearch-exporter/README.md

```diff
@@ -38,6 +38,20 @@ $ helm delete --purge my-release
 ```
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## Upgrading an existing Release to a new major version
+
+A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
+
+### To 2.0.0
+
+Update your claster to kubernetes 1.10+
```

Comment:
Would be good to have a full sentence why a certain k8s version is required

Reply:
Updated